### PR TITLE
exit code updated for command line operations

### PR DESCRIPTION
**Pull request by NehaNaithani**

### CODE REVIEW

File: iofog-agent-client/build.gradle [hunk 1]
```diff
@@ -3,9 +3,15 @@ plugins {
...
 
 description = 'iofog-agent-client'
-
+configurations {
+    // configuration that holds jars to include in the jar
+    extraLibs
+}
 dependencies {
     compile 'org.slf4j:slf4j-nop:1.7.13'
+	extraLibs ('org.apache.geronimo.specs:geronimo-json_1.0_spec:1.0-alpha-1',
```
Comment (baghbidi) [inline iofog-agent-client/build.gradle]: `https://mvnrepository.com/artifact/javax.json/javax.json-api` contains packages you need

File: iofog-agent-client/build.gradle [hunk 2]
```diff
@@ -30,12 +36,16 @@ build {
 task copyJar(type: Copy) {
     from ("$buildDir/libs/") {
         include "*-all.jar"
+
```
Comment (baghbidi) [inline iofog-agent-client/build.gradle]: Delete blank line

File: iofog-agent-client/src/main/java/org/eclipse/iofog/Client.java [hunk 3]
```diff
@@ -94,22 +98,38 @@ private static boolean sendCommandlineParameters(String... args) {
...
 				result.append(output);
 			}
-
-			conn.disconnect();
-
-			System.out.println(result.toString().replace("\\n", "\n"));
-
+			String commandLineOutput = result.toString();
+			JsonReader reader = Json.createReader(new StringReader(commandLineOutput.split("}")[0]+ "}"));
+            JsonObject commandLineJsonResponse = reader.readObject();
```
Comment (baghbidi) [inline iofog-agent-client/src/main/java/org/eclipse/iofog/Client.java]: Mixed `tabs` and `spaces` here

File: iofog-agent-client/src/main/java/org/eclipse/iofog/Client.java [hunk 4]
```diff
@@ -94,22 +98,38 @@ private static boolean sendCommandlineParameters(String... args) {
...
+            if (commandLineJsonResponse.containsKey("error")) {
```
Comment (baghbidi) [inline iofog-agent-client/src/main/java/org/eclipse/iofog/Client.java]: two blocks look the same?

File: iofog-agent-daemon/src/test/java/org/eclipse/iofog/command_line/CommandLineActionTest.java [hunk 5]
```diff
@@ -111,10 +121,15 @@ public void testGetKeys() {
     @Test
     public void testHelpActionPerform() {
         String[] helpArgs = {"help", "--help", "-h", "-?"};
-        assertEquals(helpContent, CommandLineAction.getActionByKey(helpArgs[0]).perform(helpArgs));
-        assertEquals(helpContent, CommandLineAction.getActionByKey(helpArgs[1]).perform(helpArgs));
-        assertEquals(helpContent, CommandLineAction.getActionByKey(helpArgs[2]).perform(helpArgs));
-        assertEquals(helpContent, CommandLineAction.getActionByKey(helpArgs[3]).perform(helpArgs));
+        try {
+			assertEquals(helpContent, CommandLineAction.getActionByKey(helpArgs[0]).perform(helpArgs));
```
Comment (baghbidi) [inline iofog-agent-daemon/src/test/java/org/eclipse/iofog/command_line/CommandLineActionTest.java]: mixed tabs and spaces

File: iofog-agent-client/build.gradle (hunk 1 above)
Comment (NehaNaithani) [reply iofog-agent-client/build.gradle]: updated

File: iofog-agent-client/build.gradle (hunk 2 above)
Comment (NehaNaithani) [reply iofog-agent-client/build.gradle]: updated

File: iofog-agent-client/src/main/java/org/eclipse/iofog/Client.java (hunk 4 above)
Comment (NehaNaithani) [reply iofog-agent-client/src/main/java/org/eclipse/iofog/Client.java]: removed block

**Pull request merged**